### PR TITLE
Update top banners in different bounty flow pages

### PR DESCRIPTION
**Pull request by SaptakS**
##### Description

* Fix banner text alignments and fix max-width for the banner image
* Add upper left current ballance label
* Remove bounty navigation from accept bounty and submit work pages.

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

### CODE REVIEW

File: app/dashboard/templates/shared/zero_balance_error.html [hunk 1]
```diff
@@ -20,21 +20,17 @@
...
       </p>
     </div>
     <div class="col-lg-2 col-sm-12">
       <p>
-        <a class="btn btn-sm btn-info mt-3 pulseClick" role="button" id="submit" href="{% url 'faucet' %}" target="_blank" rel="noopener noreferrer">{% trans "View Faucet" %}</a>
-        <br>
-        - {% trans "or" %} -
-        <br>
+        <a class="button button--primary mt-3 pulseClick" role="button" id="submit" href="{% url 'faucet' %}" target="_blank" rel="noopener noreferrer">{% trans "View Faucet" %}</a>
+        <br><br>
```
Comment (thelostone-mc) [inline app/dashboard/templates/shared/zero_balance_error.html]: wanna swap this out by padding `mb-2` to `<a>` ?

File: app/dashboard/templates/submit_bounty.html [hunk 2]
```diff
@@ -30,11 +30,11 @@
   </div>
 
   <div class="container-fluid">
-    <div class="row no-gutters">
-      <div class="col-sm-12 col-lg-3">
+    <div class="row no-gutters metamask-banner">
+      <div class="col-sm-12 col-lg-2">
```
Comment (thelostone-mc) [inline app/dashboard/templates/submit_bounty.html]: ❤️

File: app/dashboard/templates/shared/accept_work_banner.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      <h3>{% trans "Accepting a Submission is Exciting" %}</h3>
+      <p>
+        <ol>
+          <li>{% trans "Make sure the submitter has completed the requirements to the specifications of your bounty." %}</li>
+          <li>{% trans "Feedbacks and gratitude are always nice ways to continue a relationship." %}</li>
+        </ol>
+      </p>
+    </div>
+    <div class="col-lg-2 col-sm-12">
+      <img src="{% static "v2/images/Box.svg" %}">
```
Comment (thelostone-mc) [inline app/dashboard/templates/shared/accept_work_banner.html]: rename to `box.svg` cause I guess that's what we've been doing soo far in this repo

Comment (SaptakS) [reply app/dashboard/templates/shared/accept_work_banner.html]: Well, there are mixed namings, so I didn't refactor. If you want, I will.

File: app/assets/v2/images/Box.svg [hunk 4]
```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="97px" height="92px" viewBox="0 0 97 92" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 49 (51002) - http://www.bohemiancoding.com/sketch -->
+    <title>Metamask Copy 3</title>
```
Comment (mbeacom) [inline app/assets/v2/images/Box.svg]: We'll want to squash this with svgo.

File: app/assets/v2/images/Refund.svg [hunk 5]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="97px" height="92px" viewBox="0 0 97 92" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 49 (51002) - http://www.bohemiancoding.com/sketch -->
+    <title>Metamask Copy 4</title>
```
Comment (mbeacom) [inline app/assets/v2/images/Refund.svg]: And this one - 💃

**Comment by thelostone-mc** [comment]
@mbeacom renamed files and compressed them

**Comment by owocki** [comment]
didnt test but LGTM .

**Pull request merged**